spec/system/user_preferences_interface_spec.rb

```diff
@@ -9,6 +9,15 @@
 
   describe "Bookmarks" do
     it "changes the bookmark after notification preference" do
+      skip(<<~SQL) if ENV["CI"]
```

Comment:
The `heredoc` label is a bit odd. The text is not anything related to SQL.

Reply:
Also, long term this should be fixed. I am wondering why this is failing.

Reply:
Good catch! I've fixed the heredoc.